app/celery/tasks.py

```diff
@@ -454,3 +460,17 @@ def service_allowed_to_send_to(recipient, service):
             [user.mobile_number, user.email_address] for user in service.users
         )
     )
+
+
+def provider_to_use(notification_type, notification_id):
```

Comment:
Helper method to pick the right provider. If no active providers blow up with an exception.